cfpb/cfgov-refresh
# Increase color contrast ratio for disabled mega menu links on mobile, remove "Overview" text from disabled links on mobile

This PR fixes https://GHE/CFGOV/platform/issues/2607 by doing 2 things:

- Increases color contrast ratio for disabled mega menu links on mobile
- Keeps the text visible/readable for mobile users (including iOS VoiceOver users) in these disabled links, but removes the word "Overview" from these and repurposes them as a heading that gives mobile users additional context for the active list of links within the current level of the mobile menu

## Additions

- override the default `u-link__disabled` class color for level 2 https://snook.ca/technical/colour_contrast/colour.html#fg=5A5D61,bg=F7F8F9 links within the mega menu, to make the gray text higher contrast against the mega menu background color. This makes the text WCAG 2 AA Compliant (but not AAA) with a 6.224 color contrast ratio, according to https://snook.ca/technical/colour_contrast/colour.html#fg=5A5D61,bg=F7F8F9
![screen shot 2018-05-18 at 5 39 05 pm](https://user-images.githubusercontent.com/702526/40331305-ed7f7220-5d1d-11e8-9ba8-d51f8a3506c8.png)

## Changes

- To improve usability for mobile users navigating the mega menu, the disabled link text for these menus is kept visible on small screen sizes (both to sighted users and screen reader users). For the overview links that actually link to an overview page, the text remains the same (e.g., "Data and Research Overview"). For the 2 sections within the nav that do not currently have overview pages, the text is still not a link, has higher visual color contrast, and the word "Overview" does not appear (e.g., the heading in the mobile menu says "Consumer Tools" instead of "Consumer Tools Overview"). This text now serves as a heading that gives context to the list of links below, instead of a confusing non-active, difficult-to-read link to a non-existent overview page.
- This change was made in a way that means going forward, when additional Overview pages are active, the text will automatically update (so when Practitioner Resources Overview gets a link associated with it, the macro should work to add back in the Overview text at that time without edits to the macro code).

## Testing

1. check out branch
1. run `gulp styles` to rebuild CSS
1. test a small screen size to view the mobile mega menu and confirm the word Overview is not appearing after Consumer Tools and Practitioner Resources, but still shows in the other sections (see screenshots for how it should work below)



## Screenshots
![screen shot 2018-05-21 at 5 08 55 pm](https://user-images.githubusercontent.com/702526/40331551-c0e1cde8-5d1e-11e8-9270-e1f447adb802.png)
![screen shot 2018-05-21 at 5 08 50 pm](https://user-images.githubusercontent.com/702526/40331552-c0f4828a-5d1e-11e8-9f08-8c49f921c620.png)

![screen shot 2018-05-21 at 5 09 02 pm](https://user-images.githubusercontent.com/702526/40331550-c0ccdc58-5d1e-11e8-9bdf-8aac3c696250.png)
![screen shot 2018-05-21 at 5 09 12 pm](https://user-images.githubusercontent.com/702526/40331549-c0ba888c-5d1e-11e8-8a67-2f5b839d6970.png)
![screen shot 2018-05-21 at 5 09 16 pm](https://user-images.githubusercontent.com/702526/40331548-c0a5eb84-5d1e-11e8-8133-704ae6a0df01.png)


## Notes

-

## Todos

-

## Checklist


## Testing checklist

### Browsers


### Accessibility


### Other
